docs/DeveloperGuide.adoc

```diff
@@ -295,17 +294,58 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |teacher |add, edit, and delete questions in the word banks |make corrections on what my students are supposed to learn
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |teacher |give customised word banks and definitions |can let my students practice specific problems.
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |list all my word banks |
+
+|`* * *` |user |give titles to word banks |recognise them better
+
+|`* * *` |user |delete word banks |free up some memory when I don’t need it anymore
+
+|`* * *` |user |see the content of the word bank |study beforehand/make changes
+
+|`* * *` |young student |trivia questions to be gamified |enjoy the process
+
+|`* * *` |student |create my own question banks |tailor fit to my learning
+
+|`* * *` |computer science student |have a manual of the commands available |refer to them when I am lost
+
+|`* *` |frequent user |easily access my most recently attempted question sets |can quickly resume my revision
+
+|`* *` |studious student |set and complete goals |have something to work towards
+
+|`* *` |student |see my test statistics |track my progress/improvement
+
+|`* *` |student |choose different kinds of time constraints |can simulate exam conditions
+
+|`* *` |student |categorise my question sets |easily look for relevant materials
+
+|`* *` |student |mark question sets as important/urgent |know how to prioritise my revision
+
+|`* *` |module coordinator |export lessons |send to their students
+
+|`* *` |student |share and compare my results with my classmates |know where I stand
+
+|`* *` |student |partition the trivia |attempt questions that I'm comfortable with
```

Comment:
Consider rephrasing "partition the trivia" to "sort according to difficulty" etc..